hologram_dashboard/server/request_handler/request_handler.cc

```diff
@@ -1,62 +1,69 @@
 /*
-    Copyright 2020 Google LLC
-
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        https://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
+*   Copyright 2020 Google LLC
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       https://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
 */
+
 #include "request_handler.h"
+
 #include <vector>
 #include <sstream>
+
 using namespace Pistache;
+
 using namespace std;
 
+namespace wireless_android_play_analytics {
+
 RequestHandler::RequestHandler()
-    :server(Address(Ipv4::any(), Port(8000))) {
-        init();
+    :server_(Address(Ipv4::any(), Port(8000))) {
+        Init();
 }
 
 RequestHandler::RequestHandler(Address Addr)
-    :server(Addr) {
-        init();
+    :server_(Addr) {
+        Init();
 }
 
-void RequestHandler::init() {
+void RequestHandler::Init() {
     using namespace Rest;
-    server.init();
+    server_.init();
     
-    // Binds the router with correct request methods
-    Routes::Get(router, "/:system", 
-        Routes::bind(&RequestHandler::getDashboard, this));
-    Routes::Get(router, "/:system/:source/:date", 
-        Routes::bind(&RequestHandler::sendCommand, this));
-    Routes::Get(router, "/", 
-        Routes::bind(&RequestHandler::getLastRefreshed, this));
-
-    server.setHandler(router.handler());
+    // Binds the router with correct request methods.
+    Routes::Get(router_, "/:system", 
+        Routes::bind(&RequestHandler::GetDashboard, this));
+    Routes::Get(router_, "/:system/:source/:date", 
+        Routes::bind(&RequestHandler::SendCommand, this));
+    Routes::Get(router_, "/", 
+        Routes::bind(&RequestHandler::GetLastRefreshed, this));
+
+    server_.setHandler(router_.handler());
```

Comment:
Prefer to only keep apis for RequestHandler in this class, and move server related init and registering path to functions to server.cc.

Reply:
And add tests for RequestHandler.